src/internal/parallel/each.go

```diff
@@ -1,7 +1,12 @@
 package parallel
 
+// Action is a function that returns an error, this is used by ForEach
 type Action func() error
 
+// ForEach runs the given actions in parallel with a maximum concurrency of maxConcurrency
+// and returns a slice of errors
```

Comment:
```suggestion
// ForEach runs the given Actions concurrently with a specified maximum concurrency.
```

Motivation: no need to repeat the function signature IMO. WDYT?

Reply:
I removed the mentions of the function signature.

For now I've left the word `paralell` in there, but thats because it matches the package name. If we want to rename this later we can do in another PR.